### PR TITLE
React setup

### PR DESCRIPTION
**Pull request by Danfall369**
Hi there, Code Review

I am pleased to submit my pull request for the project requirements as outlined in the task description.

I have created a new React application using Create React App, created a reusable components directory at /components, and installed React Router V6. I have also set up the router inside of <App> and added two routes to the application: / and /categories, each of which renders a page component.

Additionally, I have created the required components inside the /components directory. The Book component contains individual book states and a button to delete the book. The BookList component renders separate books as a list. The NewBookForm component contains a form to create new books, and the Navigation component contains links to the two routes created earlier.

Please let me know if you have any feedback or questions regarding my implementation.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import ListBooks from './components/ListBooks';
+
+function App() {
+  return (
+    <Router>
+      <div className="container">
+        <NavBar />
+        <Routes>
+          <Route path="/" exact Component={ListBooks} />
+        </Routes>
```
Comment (ridwanediallo) [inline src/App.js:13]: - [ ] So far, so good. However, it is a requirement to have a component **Categories** and a **route** that leads to it. If the user clicks on the category link on the navigation bar he should be redirected to the see the **Categories** component . Please make sure to do so. :ok_hand: 
### Requirement 
Screenshot from 2023-05-06 22-29-58

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @Danfall369 , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your app is free of linters errors. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Danfall369) [reply src/App.js]: @ridwanediallo Resolve!

**Review by levy002 — APPROVED** [review]
## Hello @Danfall369 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- Implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**